packages/react-components/react-toolbar/src/components/Toolbar/useToolbar.ts

```diff
@@ -38,4 +39,33 @@ export const useToolbar_unstable = (props: ToolbarProps, ref: React.Ref<HTMLElem
     }),
     ...props,
```

Comment:
We should avoid spreading all the properties on state,  since you might be passing down the line things you might end up missing track of. You're already spreading it at `state.root` which is expected, shouldn't that be enough?

Reply:
I am trying to reproduce similar pattern introduced in the `Menu` component

Reply:
Thx for letting me know! https://github.com/microsoft/fluentui/pull/24562

Reply:
Done https://github.com/microsoft/fluentui/pull/24380/commits/55ea07ec90f6cc73c42ee5ab18d1586cd92cb906